packages/shorebird_cli/lib/src/commands/releases/list_releases_command.dart

```diff
@@ -0,0 +1,96 @@
+import 'package:collection/collection.dart';
+import 'package:intl/intl.dart';
+import 'package:mason_logger/mason_logger.dart';
+import 'package:shorebird_cli/src/code_push_client_wrapper.dart';
+import 'package:shorebird_cli/src/commands/preview_command.dart';
+import 'package:shorebird_cli/src/config/config.dart';
+import 'package:shorebird_cli/src/logger.dart';
+import 'package:shorebird_cli/src/shorebird_command.dart';
+import 'package:shorebird_cli/src/shorebird_env.dart';
+import 'package:shorebird_code_push_client/shorebird_code_push_client.dart';
+
+/// {@template list_releases_command}
+/// A command to list available releases.
+/// {@endtemplate}
+class ListReleasesCommand extends ShorebirdCommand {
+  /// {@macro list_releases_command}
+  ListReleasesCommand() {
+    argParser
+      ..addOption(
+        'flavor',
+        help: 'The product flavor to use when building the app.',
```

Comment:
```suggestion
        help: 'The flavor for which to list releases.',
```
Not sure if this is the "right" phrasing, but it's closer